src/components/ReactComponent/layout/Header/Header.tsx

```
@@ -0,0 +1,360 @@
+            {featureFlags.showNewsletter && (
+              <NavLink href="/newsletter-subscribe">Newsletter</NavLink>
+            )}
+            {featureFlags.showAbout && <NavLink href="/about">About</NavLink>}
+            {featureFlags.showProjects && (
+              <NavLink href="/project">Projects</NavLink>
+            )}
+            {featureFlags.showTags && (
+              <NavLink
+                href="/tag"
+                icon={<TagIcon />}
+                children={undefined}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)